ml_project/ml_project/enities/data_validation.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+
+
+class InputData(BaseModel):
+    id: int
+    age: int
+    sex: int
+    cp: int
```

Comment:
please, add more validations not only by type

Reply:
Data validation functions (check_data_valid and check_value_in_interval) have already been implemented in   [ml_project/inference.py](https://github.com/made-ml-in-prod-2022/FiliVan/blob/eb21887a770ae1016f9a777b0279851cc814e4bf/ml_project/ml_project/inference.py#L64)

Reply:
@nickdndev, there may be more mistakes? 
If not, could you confirm the points. I realized that I've accidentally merged and closed request